pkg/cache/cache.go

```diff
@@ -31,7 +33,7 @@ import (
 	"sigs.k8s.io/kueue/pkg/workload"
 )
 
-const workloadCapacityKey = "spec.assignedCapacity"
+const workloadClusterQueueKey = "spec.admission.clusterQueue"
```

Comment:
@alculquicondor how can we add a test for this bug?

Reply:
It actually might not be a bug, because we use the same string when setting up the index and when querying.

But this is the kind of magic in controller-runtime that I don't think we can test.

Reply:
it is not about the string though, it is about the value; the string is the key to lookup the value and it should be correct to have a correct index, otherwise what would it be indexing on?

Reply:
but we extract the value explicitly in the index

Reply:
ah, right